### DISCUSSION TITLE
Is there a OPC UA Device Service available?

### DISCUSSION THREAD

**Discussion by dmocek** [edgexfoundry/community#102] [General]
Is there a OPC UA device service that works with Jakarta?  I see the device-opcua-go project (https://github.com/edgexfoundry-holding/device-opcua-go).  Is it in a usable state?  It's in the edgexfoundry-holding area and hasn't been touched for 2 years.  Was it slated for release with jakarta?

**Comment by cloudxxx8** [comment]
we need some vonlunteers to maintain and test it before making it officially release.

**Reply by dmocek** [reply]
Does that mean you believe it's in a shippable state and it only needs some testing/bug fixing?

**Reply by cloudxxx8** [reply]
no, I've never tried that, so I can't make any evaluation.
